### DISCUSSION TITLE
Welcome to Crud Mucosa's music recommendations Discussions!

### DISCUSSION THREAD

**Discussion by crudmucosa** [crudmucosa/musicrecommendations#133] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smashwilson** [comment]
Well hello there, @crudmucosa !!!

![hello-there](https://user-images.githubusercontent.com/17565/101544320-7ad90f80-3973-11eb-8237-32685741437c.gif)

**Reply by smashwilson** [reply]
I bet everyone who built this feature is just absurdly handsome/good-looking and good at things!

**Reply by crudmucosa** [reply]
Guaranteed! I bet they also can bake a right solid pie and host a damn fine Wizard People viewing night.
